include/zephyr/xen/public/arch-arm.h

```diff
@@ -312,14 +351,26 @@ DEFINE_XEN_GUEST_HANDLE(vcpu_guest_context_t);
 
 #define XEN_DOMCTL_CONFIG_TEE_NONE     0
 #define XEN_DOMCTL_CONFIG_TEE_OPTEE    1
+#define XEN_DOMCTL_CONFIG_TEE_FFA       2
 
 struct xen_arch_domainconfig {
 	/* IN/OUT */
 	uint8_t gic_version;
+#if __XEN_INTERFACE_VERSION__ >= 0x00040e00
+	/* IN - Contains SVE vector length divided by 128 */
+	uint8_t sve_vl;
+#endif
 	/* IN */
 	uint16_t tee_type;
 	/* IN */
 	uint32_t nr_spis;
+#if __XEN_INTERFACE_VERSION__ >= 0x00041300
```

Comment:
Note: actually, there aren't these changes in mainline
it is EPAM related: https://github.com/xen-troops/xen/commit/d4192ca563d80ca775eadbce278eccc3ebc1bc8e#diff-f7cea1bef753331288b1f2c3e15a1d9a6f6e4e48ef59cfabff177d185c8503a8R343